Changes

```diff
@@ -2,6 +2,26 @@
 Revision history for Perl extension Mojo-IOLoop-ReadWriteProcess
 
 {{$NEXT}}
+0.35 2025-03-17 12:20:20Z
```

Comment:
Can we use a modern version here?
```suggestion
v0.350 2025-03-17 12:20:20Z
```
https://github.com/perlpunk/perl5-module-meta links to
https://blogs.perl.org/users/grinnz/2018/04/a-guide-to-versions-in-perl.html

Reply:
no, we should use `v0.350.0`.
v0.35.0 would be lower than 0.35, beause 0.35 is the same as 0.350

The git tag is completely irrelevant to CPAN.

I'm not sure what you mean by linking to https://build.opensuse.org/projects/devel:languages:perl/packages/perl-Perl-Tidy/files/perl-Perl-Tidy.spec?expand=1
That's what cpanspec makes of the version from CPAN. It removes the `v` from the beginning for rpm.

So: please use the `v` everywhere and two dots.

Reply:
https://metacpan.org/dist/YAML-PP
Tarball name: YAML-PP-v0.39.0.tar.gz

https://metacpan.org/release/TINITA/YAML-PP-v0.39.0/source/META.json#L214
```
"version" : "v0.39.0",
```
https://metacpan.org/release/TINITA/YAML-PP-v0.39.0/source/META.yml
```
version: v0.39.0
```
https://metacpan.org/release/TINITA/YAML-PP-v0.39.0/source/Changes#L3
```
v0.39.0 2025-02-10 00:01:45+01:00
```
https://metacpan.org/dist/YAML-PP/source/lib/YAML/PP.pm#L6
```
our $VERSION = 'v0.39.0'; # VERSION
```

Reply:
and https://blogs.perl.org/users/grinnz/2018/04/a-guide-to-versions-in-perl.html
> For this reason it's best to include both a leading "v" and at least two decimal separators for clarity when using tuple versions.

Reply:
And linking to https://github.com/search?q=repo%3Amojolicious%2Fmojo%209.40&type=code is irrelevant, because Mojolicious still uses the decimal version format.

Reply:
as it says "when using tuple versions" but I wouldn't use tuple versions but decimal version format. Again, I would avoid the ambiguity about 0.35X by using 1.0.0

Reply:
-> https://github.com/openSUSE/Mojo-IOLoop-ReadWriteProcess/pull/74

Reply:
> as it says "when using tuple versions" but I wouldn't use tuple versions but decimal version format. Again, I would avoid the ambiguity about 0.35X by using 1.0.0

1.0.0 is a tupled version. and my suggestion was to use the tupled version format from now on.